src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -519,9 +519,9 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
             {
                 return false;
             }
-            else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
+            else if (RuntimeEnvironmentHelper.IsFreeBSD || RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
```

Comment:
```suggestion
            else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX || RuntimeEnvironmentHelper.IsFreeBSD)
```

Since C# does lazy evaluation, in order, it's better to put the most likely expression first, least likely expression last, to minimize work.

Reply:
Since I removed the `RuntimeEnvironmentHelper.IsFreeBSD` property again, this is no longer relevant.